portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSet.testcase

```diff
@@ -1107,6 +1107,577 @@ definition {
+                locator1 = "FormViewBuilder#OBJECT_FIELD_BUTTON_DROPDOWN");
+
+            Type(
+                key_text = "Name",
+                locator1 = "TextInput#ANY",
+                value1 = "Bike");
+
+            Click(
+                key_class = "btn-secondary",
+                key_index = 2,
+                locator1 = "Button#ANY_WITH_CLASS_AND_INDEX");
+
+            SelectFrameTop();
+
+            Click.javaScriptClick(
+                key_sortOption = "Relationship_c_fdsViewERC",
+                locator1 = "ObjectCustomViews#SORT_OPTION");
+
+            Select(
+                locator1 = "DataSet#SOURCE_OPTIONS_SELECT",
+                value1 = "Transport");
+        }
+
+        task ("And Preselected Values bike") {
+            Click(
+                key_fieldName = "Select a default value for your filter.",
+                locator1 = "ObjectCustomViews#FROM_DATE_INPUT");
+
+            Click(
+                key_sortOption = "bike",
+                locator1 = "ObjectCustomViews#SORT_OPTION");
+        }
+
+        task ("And Select the option Multiple,Include radio button") {
+            Click.clickAtNotVisible(
+                key_radioOption = "Multiple",
+                locator1 = "CommerceEntry#RADIO_BUTTON");
+
+            Click.clickAtNotVisible(
+                key_radioOption = "Include",
+                locator1 = "CommerceEntry#RADIO_BUTTON");
+        }
+
+        task ("And the user save the change clicking on save button") {
+            Button.clickSave();
+        }
+
+        task ("When the user adds a widget page and add a fragment Data Set") {
+            Navigator.openURL();
+
+            ContentPages.addPage(pageName = "Test Page");
+
+            PageEditor.addFragment(
+                collectionName = "Content Display",
+                fragmentName = "Data Set");
+        }
+
+        task ("And viewing the configuration menu of the portlet And selects View Fields Test") {
+            Click(locator1 = "ReportsAdminViewReportEntry#INFO_MESSAGE");
+            
+			Click.javaScriptClick(
+                key_value = "Select Data Set View",
+                locator1 = "Button#SECONDARY_BY_TITLE");
+            
+			SelectFrame(
+                key_title = "Select",
+                locator1 = "IFrame#MODAL_ANY");
+            
+			Click.clickAtNotVisible(
+                key_radioOption = "View Fields Test",
+                locator1 = "CommerceEntry#RADIO_BUTTON");
+            
+			Button.clickSave();
+        }
+
+        task ("And publishes the page") {
+            SelectFrame(locator1 = "DataSet#DEFOCUS");
+
+            Click.javaScriptClick(locator1 = "WorkflowMetricsIndex#METRICS_REINDEX_ALL_BUTTON");
+        }
+
+        task ("Then the data set fragment pagination is according to the default configuration") {
+
+            Navigator.openURL();
+
+            Navigator.gotoPage(pageName = "Test Page");
+        }
+    }
```

Comment:
Here you are saying that you are checking the pagination but:

You don't have to check the pagination, you have to check that filter is working
In the task you are not checking anything, you are just going to the page
Please add the assert checking the filter is present and working (Exclude option is selected and the preselected values are present)